src/misc/cache/lv_cache.h

```diff
@@ -0,0 +1,68 @@
+/**
+* @file lv_cache.h
+*
+*/
+
+
+#ifndef LV_CACHE1_H
+#define LV_CACHE1_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "_lv_cache_internal.h"
```

Comment:
```suggestion
#include "lv_cache_private.h"
```
"private" is already used by `lv_display/indev_private.h`.